diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/IdentifierNaming.kt

```diff
@@ -141,6 +143,11 @@ class IdentifierNaming(private val configRules: List<RulesConfig>) : Rule("ident
                     if (variableName.text.containsOneLetterOrZero() && variableName.text.length > 1) {
                         VARIABLE_NAME_INCORRECT.warn(configRules, emitWarn, isFixMode, variableName.text, variableName.startOffset)
                     }
+                    // check if identifier of a property has a confusing name
+                    if (CONFUSING_IDENTIFIER_NAMES.contains(variableName.text) && !validCatchIdentifier(variableName)
+                            && node.elementType == ElementType.PROPERTY) {
+                        warnConfusingName(variableName)
```

Comment:
Do we need to check the right side of property to find expected value @acheshkov?

Reply:
good point, but by for now diktat is not able to work with such tricky things, so let's check only left side by for now